fennel/internal_lib/to_proto/source_code.py

```diff
@@ -210,11 +282,96 @@ def lambda_to_python_regular_func(lambda_func):
         return None
 
     lambda_text = dedent(
-        source_text[lambda_node.col_offset : lambda_node.end_col_offset]
+        get_text_between_ast_pointers(
+            source_text,
+            lambda_node.lineno,
+            lambda_node.col_offset,
+            (
+                lambda_node.end_lineno
+                if lambda_node.end_lineno
+                else lambda_node.lineno
+            ),
+            (
+                lambda_node.end_col_offset
+                if lambda_node.end_col_offset
+                else lambda_node.col_offset
+            ),
+        )
     )
+    lambda_next_line(lambda_func, lambda_text, line_num)
     return lambda_text
 
 
+def lambda_next_line(lambda_func: Callable, lambda_text: str, line_num: int):
+    """
+    This function checks if next_line after line_num can be included in the lambda or not.
+    If yes then throw an error if not then return
+    """
+    # Get the next line and append to lambda text
+    next_line = ""
+    with open(lambda_func.__code__.co_filename, "r") as f:
+        lines = f.readlines()
+        if line_num < len(lines):
+            next_line = lines[line_num].strip()
+    # No Next line found
+    if next_line == "":
+        return
+    # Next line already in lambda_text
+    if next_line in lambda_text:
+        return
+    # If Next line is part on next operator
```

Comment:
this is just an optimization right ? I can totally see folks adding a new operator totally forgetting to add it here. 
One thing we could do is find all functions defined on _Node class and return early using that

Reply:
This was a needed thing, what was happening was something like event.filter(...) and if the next operator was coming in new line, that was getting appended to tha lambda function properly if the lambda expression was in a parenthesis. So lambda used to become like lambda x: (x["fdgd"] > 23).assign() or any filter operator and this to used to cause/return ambiguous lambda error. But yeah makes sense to fetch all the operators through some operation in _Node class.